ubuntu/ubuntu-18.x/ubuntu-18.04-hpc/install_mpis.sh

```diff
@@ -1,5 +1,117 @@
 #!/bin/bash
 set -ex
 
-# Install common MPIs for Ubuntu
-$UBUNTU_COMMON_DIR/install_mpis.sh 18.04 "4fc27012dd9f359c919ee4e681a8a41a5d5a467f40fa89b95f26b7a4106bf1b9"
+# Load gcc
+set CC=/usr/bin/gcc
+set GCC=/usr/bin/gcc
+
+INSTALL_PREFIX=/opt
+
+# HPC-X v2.13.1
```

Comment:
Any reason we are are using 2.13 here instead of 2.14?

Reply:
Because it is Ubuntu 18.04. I thought we are only looking into 20.04 and 22.04. Should we update the 18.04 as well?

Reply:
got it .. thanks .. we can update it in the next revision.